homeworks/rostyslav.vyshemirskyi_Rooosss/PokemonProject/login.html

```diff
@@ -37,21 +37,22 @@
                 <section class="login">
                     <div class="login__inner">
                         <h1 class="login__title">Login page</h1>
-                        <form class="login__form" action="success.html">
+                        <form class="login__form" action="">
```

Comment:
Hey, @Rooosss, better to remove emty from `action` https://html.spec.whatwg.org/#attr-fs-action

Reply:
Hey, @Rooosss, please don't resolve the comment, I do it after check your fixes. 
Better add a comment `fixed`